integreat_cms/cms/migrations/0056_update_observer_permissions.py

```diff
@@ -0,0 +1,47 @@
+from django.db import migrations
+
+ROLES = [
+    {
+        "name": "EDITOR",
+        "remove_permissions": [
+            "replace_mediafile",
+        ],
+    },
+]
```

Comment:
Hmm, I see that you tried to optimize the "typical" role-update-migration a bit, but this has the downside of including one potential source of errors... in this case, you used "EDITOR" instead of "OBSERVER", and instead of fixing it, I would just suggest to use our generic approach for the role update... this way, we only have to define our permissions once in the constants and don't have to think about them again in the migrations... we just apply the config from the constants.

Reply:
Oh, what a leathal mistake 😨 

Do you mean a migration like 0043, taking the constants and updating all the permissions and roles?

Reply:
Yes, exactly! :+1:

Reply:
Done 👍